cmd/server/transport/http/message.go

```diff
@@ -125,3 +125,10 @@ func (h *Handler) GetMessage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func (h *Handler) messageRoutes() {
+	h.Router.HandleFunc("/api/v1/message", JWtAuth(h.PostMessage)).Methods("POST")
+	h.Router.HandleFunc("/api/v1/message/{id}", h.GetMessage).Methods("GET")
+	h.Router.HandleFunc("/api/v1/message/{id}", JWtAuth(h.DeleteMessage)).Methods("DELETE")
+	h.Router.HandleFunc("/api/v1/message/{id}", JWtAuth(h.UpdateMessage)).Methods("PUT")
```

Comment:
Questions:
1. How do `h` knows which function would be triggered (e.g. on h.PostMessage, h.GetMessage, etc.)?
2. When to use parenthesis? Seems like `h.PostMessage` points to `internal/database/message.go`'s > `PostMessage()` function seamlessly.